src/applications/debt-letters/components/DebtLettersWrapper.jsx

```diff
@@ -13,15 +12,17 @@ const DebtLettersWrapper = ({
   showDebtLetters,
   isProfileUpdating,
   isLoggedIn,
-  getDebtLetters,
 }) => {
+  const dispatch = useDispatch();
+
   useEffect(
     () => {
       if (showDebtLetters) {
-        getDebtLetters();
+        const generateGetDebtLetters = () => fetchDebtLetters(dispatch);
+        dispatch(generateGetDebtLetters);
```

Comment:
Is there a reason you can't just do this:
```
dispatch(fetchDebtLetters);
```

Reply:
Turns out there was not! I was unaware of that syntax and have updated the code to reflect the suggestion! The tests still passed with that new dispatch in place.

Reply:
Great, glad that worked!